test/specs/scenarios/face.js

```diff
@@ -44,6 +45,12 @@ export const faceScenarios = (lang) => {
 
     const copy = basePage.copy(lang)
 
+    async function takePercySnapshotWithoutOverlay(driver, text) {
```

Comment:
Why is this needed?

Reply:
So, we have some tests where there is a 'video' overlay placed by the tests, and because of the way the screenshots are taken by Percy they obstruct the view. This method will take the screenshot and ignore the overlay...

Reply:
Its so we DONT see something like this captured...
![Screenshot 2021-05-24 at 18 14 24](https://user-images.githubusercontent.com/79530996/119383984-f3325380-bcbb-11eb-9439-b9b572e145b2.png)
But we see...
![Screenshot 2021-05-24 at 18 15 48](https://user-images.githubusercontent.com/79530996/119384097-178e3000-bcbc-11eb-98ed-9a433e9df4ca.png)